src/app/views/authentication/Authentication.tsx

```diff
@@ -1,182 +1,152 @@
-import hello from 'hellojs';
 import React, { Component } from 'react';
-
 import { connect } from 'react-redux';
 import { bindActionCreators, Dispatch } from 'redux';
-
 import { IAuthenticationProps, IAuthenticationState } from '../../../types/authentication';
 import * as authActionCreators from '../../services/actions/auth-action-creators';
 import * as queryActionCreators from '../../services/actions/query-action-creators';
-import { ADMIN_AUTH_URL, AUTH_URL, DEFAULT_USER_SCOPES , TOKEN_URL,
-  USER_INFO_URL, USER_PICTURE_URL } from '../../services/constants';
+import { USER_INFO_URL, USER_PICTURE_URL } from '../../services/constants';
 import SubmitButton from '../common/submit-button/SubmitButton';
 import './authentication.scss';
+import { getAccessToken, logOut } from './AuthService';
 import { Profile } from './profile/Profile';
 
 export class Authentication extends Component<IAuthenticationProps,  IAuthenticationState> {
-  public state = {
-    authenticated: {
-      status: false,
-      user: {
-        displayName: '',
-        emailAddress: '',
-        profileImageUrl: '',
+  constructor(props: any) {
+    super(props);
+    this.state = {
+      authenticatedUser: {
+        status: false,
+        user: {
+          displayName: '',
+          emailAddress: '',
+          profileImageUrl: '',
+        },
+        token: '',
       },
-      token: '',
-    },
-    loading: false,
-  };
+      loading: false,
+    };
+  }
 
   public componentDidMount = () => {
-    const authenticated = localStorage.getItem('authenticated');
-    if (authenticated && this.props.actions && JSON.parse(authenticated).status) {
-      this.props.actions.authenticateUser(JSON.parse(authenticated));
+    const authenticatedUser = localStorage.getItem('authenticatedUser');
+    if (authenticatedUser && this.props.actions && JSON.parse(authenticatedUser).status) {
+      this.props.actions.authenticateUser(JSON.parse(authenticatedUser));
       this.setState({
-        authenticated: JSON.parse(authenticated),
+        authenticatedUser: JSON.parse(authenticatedUser),
```

Comment:
JSON.parse is being called multiple times in this function. If you make the return value a variable you will only have to call it ones.

```
const authUser = JSON.parse(authenticatedUser)
```